src/test/java/core/basesyntax/service/FileWriterServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterServiceTest {
+    private FileWriterService writerServiceTest;
+    private ReportService reportTest;
+
+    @Before
+    public void setUp() {
+        writerServiceTest = new FileWriterServiceImpl();
```

Comment:
services should be initialized in BeforeClass, because we don't need to do this before each test
same everywhere